bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +22,279 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=1,
+                        class_naming_camel_case: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        enforce_newline_at_EOF: bool=False,
+                        use_spaces: bool=True,
+                        tab_width: int=2,
+                        number_of_newlines_after_classes: int=3,
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=False,
+                        no_implicit_parentheses: bool=False,
+                        no_interpolation_in_single_quotes: bool=True,
```

Comment:
Hm let's go with default here which allows string interpolation in single quotes.